defender/account-low-on-gas/autotasks/accountBalanceCheck/index.ts

```diff
@@ -0,0 +1,50 @@
+import { AutotaskEvent, SentinelConditionResponse } from 'defender-autotask-utils';
+const { DefenderRelayProvider } = require('defender-relay-client/lib/ethers');
+import { ethers } from 'ethers';
+
+const ScopedSecretsProvider = function (event: any) {
+  const STACK_DELIM = '_';
+  const namespace = event.autotaskName as any;
+  const key = function* (name: string) {
+    const arr = namespace.split(STACK_DELIM).concat(name);
+    do yield arr.join(STACK_DELIM);
+    while (arr.splice(-2, 2, name).length > 1);
+  };
+  const find = (name: any, target: any) => {
+    for (var i of key(name)) if (i in target) return target[i];
+  };
+  return new Proxy(event.secrets, { get: (target, name) => find(name, target) });
+};
+
+export async function handler(event: AutotaskEvent) {
+  const match = event?.request?.body as any;
+  const provider = new DefenderRelayProvider(event);
+  const scopedSecrets = ScopedSecretsProvider(event);
+
+  const _threshold = scopedSecrets[`LOW_GAS_TRESHOLD`];
```

Comment:
consider changing `LOW_GAS_TRESHOLD` to `LOW_GAS_THRESHOLD` and replacing backticks